examples/quickstart_tensorflow/README.md

```diff
@@ -0,0 +1,66 @@
+# Flower Example using Keras
+
+Flower is easy to use by only starting the Flower server and the Flower clients.
+
+It is recommended to use [pyenv](https://github.com/pyenv/pyenv)/[pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) and [poetry](https://python-poetry.org/docs/) to ensure the right version of libraries.
+
+After installing both tools you can start to set up everything.  
```

Comment:
```suggestion
This example uses [poetry](https://python-poetry.org) for dependency management. If not checkout the [poetry](https://python-poetry.org) information howto install it.
```

Reply:
The changes will be added in the next PR